build-selftests/build_selftests.sh

```diff
@@ -55,6 +55,8 @@ SELF_OPTS=$(cat <<EOF
 	-C ${REPO_ROOT}/${REPO_PATH}/tools/testing/selftests/bpf
 EOF
 )
+export LLVM_CONFIG=llvm-config-${LLVM_VERSION}
```

Comment:
what is relying on this? this looks a bit random

Reply:
`tools/build/feature/Makefile` uses this variable